README.md

```diff
@@ -1 +1,18 @@
-# proposals
\ No newline at end of file
+# Proposals
+This is an index of the proposed changes in _Bazel embedded_ ecosystem. 
+
+## Index
+
+### Under Review
+| Created | Title | Authors(s) | Category |
+|--------------|-------|------------|----------|
+
+### Draft
```

Comment:
What's the difference between a proposal that's a draft vs. under review? (I'm wondering whether these sections are redundant.

Reply:
That's true, we can probably leave the drafts section out. I'm vaguely using [bazelbuild/proposals](https://github.com/bazelbuild/proposals) as a template. Though I think you are right in that it doesn't make much sense.

Reply:
Oh neat! Sounds good 👍 Maybe worth referencing that repo in this README along with the intended design process (assuming it will be similar)?